modules/openapi-generator/src/main/resources/typescript-node/models.mustache

```diff
@@ -68,6 +68,16 @@ let typeMap: {[index: string]: any} = {
     {{/models}}
 }
 
+// Check if a string starts with another string without using es6 features
+function startsWith(str: string, match: string): boolean {
+    return str.lastIndexOf(match, 0) === 0;
+}
+
+// Check if a string ends with another string without using es6 features
+function endsWith(str: string, match: string): boolean {
+    return str.length > match.length && str.substring(str.length - match.length) === match;
```

Comment:
Good catch. It would be fine without it because all of the matches being passed in are strict subsets (there is guaranteed to be more string than match), but still good to be correct!